lib/api/api.js

```diff
@@ -271,6 +344,15 @@ const api = {
                     return acc;
                 }, {});
             }
+            if (apiMethod === 'objectPost') {
+                request._response = response;
+                if (fileEventData) {
+                    request.fileEventData = fileEventData;
+                    request.headers['content-type'] = fileEventData.mimetype;
```

Comment:
AWS doesn't use the mimetype of the file as they have their other field `Content-Type` used to set the content type of the file, by default they leave it to `application/octet-stream`

Reply:
I saw on aws it was `binary/octet-stream`

added
https://github.com/scality/cloudserver/pull/5601/files#diff-69284c3576c9c4651c9ab95928cbb676d75009ef4ee5becb238b45e22bb20efeR158